resources/lang/en/Template.properties

```diff
@@ -59,6 +59,7 @@ basketCheckout = "Checkout"
 basketContent = "Content"
 basketCoupon = "Coupon"
 basketDelete = "Delete"
+basketExportDeliveryWarning = ":to :from"
```

Comment:
basketExportDeliveryWarning = "By placing the order I recognise that my order is shipped from :from to :to. The local VAT, costs of customs clearance and customs duties are not included in the total amount of the order and have to be settled with third parties. I bear the costs."